signature-aggregator/aggregator/aggregator.go

```diff
@@ -343,6 +343,8 @@ func (s *SignatureAggregator) CreateSignedMessage(
 	return nil, errNotEnoughSignatures
 }
 
+// TODO: consider making this function private. its only reference seems to be
```

Comment:
Yup both of the below should be private. That was my oversight in the move

Reply:
addressed in https://github.com/ava-labs/awm-relayer/pull/457/commits/e5d44b441029b356278d6d13549ff6c4fdb89291